src/ble/att/handle.rs

```diff
@@ -21,6 +21,11 @@ impl AttHandle {
     pub fn as_u16(&self) -> u16 {
         self.0
     }
+
+    /// Checks if an AttHandle is in a given range
+    pub fn check(&self, range: RangeInclusive<AttHandle>) -> bool {
```

Comment:
Not too happy about the name `check`, it doesn't really tell you anything. Maybe we should add a `HandleRange` type that has a `contains` method, similar to the unstable `contains` method on Rust's ranges?

Reply:
Done :)